recipes-kernel/kernel-tests/kernel-tests-files/kernel_dmesg_diff.py

```diff
@@ -221,14 +221,31 @@ def strip_timestamps(log):
         [r'clocksource: tsc: mask: 0x[0-9a-fA-F]+ max_cycles: 0x[0-9a-fA-F]+, max_idle_ns: \d+ ns', 'clocksource: tsc: mask: 0x max_cycles: 0x, max_idle_ns: ns']
 ]
 
-def strip_known_differences(log):
+def apply_replacement_patterns(log):
     for pattern in replacement_patterns:
         log = re.sub(pattern[0], pattern[1], log, flags=re.MULTILINE)
     return log
 
+known_intermittent_messages = [
+    [r'i915 0000:00:02.0: \[drm\] Cannot find any crtc or sizes'],
```

Comment:
I think we should ignore this for any PCI bus address not just '0000:00:02.0:'. The address is unlikely to change on the current test system but may fail in the future if we add coverage for different hardware.